### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

File: web/ralucarbc/html/basic/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
```
Comment (marianzburlea) [inline web/ralucarbc/html/basic/index.html]: add at least one tag with content into the body

**Review by marianzburlea — CHANGES_REQUESTED** [review]

File: web/ralucarbc/html/basic/index.html (hunk 1 above)
Comment (RalucaRbc) [reply web/ralucarbc/html/basic/index.html]: IT HAS BEEN UPDATED

**Review by marianzburlea — APPROVED** [review]
💯

**Review by biggicode — APPROVED** [review]

**Review by BogdanNegura — APPROVED** [review]
:)

**Pull request merged**